gtfs-realtime/spec/en/reference.md

```diff
@@ -186,6 +187,7 @@ The update is linked to a specific stop either through stop_sequence or stop_id,
 | **stop_id** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Conditionally required | One | Must be the same as in stops.txt in the corresponding GTFS feed. Either stop_sequence or stop_id must be provided within a StopTimeUpdate - both fields cannot be empty. If `StopTimeProperties.assigned_stop_id` is populated, it is preferred to omit `stop_id` and use only `stop_sequence`. If `StopTimeProperties.assigned_stop_id` and `stop_id` are populated, `stop_id` must match `assigned_stop_id`. |
 | **arrival** | [StopTimeEvent](#message-stoptimeevent) | Conditionally required | One | If schedule_relationship is empty or SCHEDULED, either arrival or departure must be provided within a StopTimeUpdate - both fields cannot be empty. arrival and departure may both be empty when schedule_relationship is SKIPPED.  If schedule_relationship is NO_DATA, arrival and departure must be empty. |
 | **departure** | [StopTimeEvent](#message-stoptimeevent) | Conditionally required | One | If schedule_relationship is empty or SCHEDULED, either arrival or departure must be provided within a StopTimeUpdate - both fields cannot be empty. arrival and departure may both be empty when schedule_relationship is SKIPPED.  If schedule_relationship is NO_DATA, arrival and departure must be empty. |
+| **departure_occupancy_status** | [OccupancyStatus](#enum-occupancystatus) | Required | One | The state of passenger occupancy for the vehicle immediately after departure from the given stop. |
```

Comment:
I see CrowdLevel vs OccupancyStatus as a separate issue that should be resolved by #261. I think we can move ahead with predicted occupancy using OccupancyStatus at this time and then add CrowdLevel once #261 is hashed out. I don't see us removing OccupancyStatus as part of #261 so I think it would be acceptable to also have two values for predicted occupancy. At that time I would also like to see occupancy_percentage be discussed as a possible part of predicted occupancy since the enumerated types are likely too limited to be of universal use.

Reply:
Amended for the first 3 comments above.

For `CrowdLevel` vs `OccupancyStatus`, I'd prefer to separate that out from this and use `OccupancyStatus` here since that's currently used in `VehiclePosition`, then update this when/if VP changes.